app/models/katello/glue/pulp/node.rb

```diff
@@ -0,0 +1,22 @@
+module Katello
+  module Glue::Pulp::Node
+    def self.included(base)
```

Comment:
I would probably just stick this in smart_proxy_extensions itself, we're slowly moving away from a glue structure of things.